packages/plugin-core/src/test/suite-integ/SyncCommand.test.ts

```diff
@@ -7,24 +7,45 @@ import { GitTestUtils } from "@dendronhq/engine-test-utils";
 import { tmpDir } from "@dendronhq/common-server";
 import { describe } from "mocha";
 import { NoteTestUtilsV4 } from "@dendronhq/common-test-utils";
+import { getWS } from "../../workspace";
+import { Git } from "@dendronhq/engine-server";
+import { DendronConfig } from "@dendronhq/common-all";
 
 suite("workspace sync command", function () {
   let ctx: vscode.ExtensionContext;
   ctx = setupBeforeAfter(this, {});
 
+  describe("no repo", () => {
+    test("do nothing", (done) => {
+      runLegacyMultiWorkspaceTest({
+        onInit: async ({}) => {
+          const out = await new SyncCommand().execute();
+          expect(out).toBeTruthy();
+          const { committed, pulled, pushed } = out as any;
```

Comment:
instead of an `any` cast, can you use the `!` operator so that the output has stricter types?